app/build.gradle

```diff
@@ -3,7 +3,7 @@ apply plugin: 'com.google.gms.google-services'
 
 android {
     compileSdkVersion 23
-    buildToolsVersion "23.0.1"
+    buildToolsVersion "23.0.2"
```

Comment:
HIer liegt das Problem.
Bitte hier die Änderungen zurücknehmen oder in `.travis.yml` ebenfalls entsprechend ändern.